### ISSUE TITLE
No-JS/No-Footer pages a11y bug

### ISSUE THREAD

**Issue by rcantin-w** [wellcomecollection/wellcomecollection.org#8717]
A test has just started failing in pa11y on [this page](https://wellcomecollection.org/works/e7vav3ss/items?page=1&canvas=1&sierraId=b28136615&langCode=eng):

![Image](https://user-images.githubusercontent.com/110461050/196919617-8901967c-b4d6-4aa7-9a0f-3045bab4c108.png)

A few things:
1. This link should only be usable when Javascript is disabled + on mobile;
2. Without a footer on a page, the user has no access to a menu. They would have to click the logo and then try the hamburger menu again.
3. Item Viewer pages are currently blank with JS disabled; there should be a no-js version available and we don't know why it's not showing.

We discussed a few solutions for the footer issue 
1. Always show it when JS is not enabled
2. Refactor it to work without JS (using [details](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details)?)

As for the Item Viewer, an investigation would be good to figure out why it's just blank at the moment.

**Comment by davidpmccormick** [comment]
Fixed in #9688
